netcore/CoreFX.issues.rsp

```diff
@@ -175,16 +175,6 @@
 # https://github.com/mono/mono/blob/18f0bf055eb2bfa8981062bdad3af2fb4bbc9be4/netcore/System.Private.CoreLib/shared/System/String.cs#L811-L821
 -nomethod System.Tests.StringTests.NormalizationTest
 
-# Environment.FailFast is not implemented
```

Comment:
These tests are unrelated actually - they call FailFast because of Debug.Assert (in debug mode)
Something wrong with fp operations inside number formatting

Reply:
Interesting. I am pretty sure they passed when I removed the exclusions, perhaps for unrelated reasons.

Reply:
```
  Discovering: System.Memory.Tests (method display = ClassAndMethod, method display options = None)
  Discovered:  System.Memory.Tests (found 2295 of 2347 test cases)
  Starting:    System.Memory.Tests (parallel test collections = on, max threads = 4)
  Finished:    System.Memory.Tests
=== TEST EXECUTION SUMMARY ===
   System.Memory.Tests  Total: 45383, Errors: 0, Failed: 0, Skipped: 0, Time: 26.037s
```

Reply:
Did you run them in debug mode? (`make debug-bcl && MONO_ENV...`)

Reply:
Nope... good point.

Reply:
I can reproduce it now. I'll put the exclusions back and clarify the comment. Thanks!